### PR TITLE
[tf] update utility node requirement

### PR DESCRIPTION
**Pull request by rustielin**
### Description
bump the instance size for utilities to 8vcpu 32gb memory so we can recommend beefier HAProxy deployments

### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4404)

### CODE REVIEW

File: terraform/aptos-node-testnet/variables.tf [hunk 1]
```diff
@@ -164,12 +164,12 @@ variable "validator_instance_max_num" {
 
 variable "utility_instance_type" {
   description = "Instance type used for utilities"
-  default     = "t3.medium"
+  default     = "t3.2xlarge"
```
Comment (sherry-x) [inline terraform/aptos-node-testnet/variables.tf:167]: have you checked if t3 is still the best instance type to use in this case?

Comment (rustielin) [reply terraform/aptos-node-testnet/variables.tf:167]: General purpose cost-per-performance, yea it seems so.

**Review by sherry-x — APPROVED** [review]
update for Azure as well?

**Comment by rustielin** [comment]
@sherry-x the Azure utilities group size for some reason is `Standard_B8ms`, which is 8 vcpu 32 gb memory already!

**Pull request merged**